media/js/src/forms/SelectionForm.jsx

```diff
@@ -173,10 +174,11 @@ export default class SelectionForm extends React.Component {
 
                 {hasVocab && (
                     <Form.Group className="mb-3">
-                        <label htmlFor="react-select-7-input">
+                        <label htmlFor="newSelectionTerms">
                             Course Vocabulary</label>
                         <Select
-                            id="newSelectionTerms"
+                            id="new-Selection-Terms"
+                            inputId='newSelectionTerms'
```

Comment:
This is minor but there is some inconsistency with the capitalization between here and the other inputs being changed in this PR. In addition, just using a different casing convention to differentiate ID's doesn't really feel semantic or meaningful - it could get confusing. I propose we stick to either dashes, or camelCase with these changes, but not both.

To differentiate between the different elements `id` and `inputId` are referring to, how about going with the convention of:
```
<Select
    id="new-selection-terms-selectcontainer"
    inputId="new-selection-terms-input"
```
This way it's self-explanatory and more obvious what's going on. `id` is really referring to react-select's container, according to the docs - https://react-select.com/props